lib/icons/sync.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+export default function SyncIcon() {
+  return (
+    <svg
+      className="icon-sync"
+      xmlns="http://www.w3.org/2000/svg"
+      width="18px"
+      height="12px"
+      viewBox="0 0 18 12"
+    >
+      <path d="M17.625,7.125 L15.3105,9.4395 L14.25,10.5 L10.875,7.125 L11.9355,6.0645 L13.5,7.629 L13.5,6 C13.5,3.519 11.481,1.5 9,1.5 L9,0 C12.3135,0 15,2.6865 15,6 L15,7.629 L16.5645,6.0645 L17.625,7.125 Z M4.5,6 L4.5,4.371 L6.0645,5.9355 L7.125,4.875 L3.75,1.5 L2.6895,2.5605 L0.375,4.875 L1.4355,5.9355 L3,4.371 L3,6 C3,9.3135 5.6865,12 9,12 L9,10.5 C6.519,10.5 4.5,8.481 4.5,6 Z" />
+    </svg>
+  );
+}
```

Comment:
Where is this icon from? Is it a GridIcon?

Reply:
I'm also wondering if we should show this icon. It may look like a button to some that you can click to start a sync?

Reply:
Yes, it's from the Gridicons set.
https://github.com/Automattic/gridicons/blob/master/sources/svg/gridicons-sync.svg

> I'm also wondering if we should show this icon. It may look like a button to some users that you can click to start a sync?

I'm a little conflicted on this, too. The button thing is definitely a concern, but I also feel like it's harder to understand what the indicator even is without the sync icon. (It's less of a problem in the native macOS app, since the Navigation Bar is always visible, and we don't have to think about the webapp case where the Settings button and footer links are in close proximity.)

One mitigating factor is that the sync icon is only shown when the websocket is connected. So in most situations, a user would only see that sync icon in conjunction with "All changes synced" — thus no reason to click it.

Reply:
> One mitigating factor is that the sync icon is only shown when the websocket is connected. So in most situations, a user would only see that sync icon in conjunction with "All changes synced" — thus no reason to click it.

That makes sense, let's leave it in then and I'll add it to macOS :)